microprofile/reactive-streams-operators-smallrye/cdi-provider/pom.xml

```diff
@@ -40,6 +40,18 @@
     <artifactId>wildfly-microprofile-reactive-streams-operators-cdi-provider</artifactId>
     <name>WildFly: MicroProfile Reactive Streams Operators CDI Provider</name>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>wildfly-legacy-ee-bom</artifactId>
```

Comment:
Shouldn't this be wildfly-legacy-expansion-bom instead?

Reply:
In any case, since PR Jobs are in green, it means this module doesn't need any expansion dependencies, so it should not prevent merging this, so LGTM